pyproject.toml

```
@@ -12,16 +12,15 @@ license = "MIT"
 name = "actfw-jetson"
 readme = "README.md"
 repository = "https://github.com/Idein/actfw-jetson"
-version = "0.1.0"
+version = "0.1.1"
 
 [tool.poetry.dependencies]
 python = "^3.6"
 
 # See comment in https://github.com/Idein/actfw-core/blob/master/pyproject.toml .
 Pillow = "^5"
 PyGObject = "^3.40.1"
-actfw-core = "^1.5.2"
-actfw-gstreamer = "*"
+actfw-core = "^2.0.0"
```

If you mean this update enables users to use a USB camera, is the following correct?

```suggestion
actfw-core = "^2.1.1"
```

---

Thank you, I also think it's more preferable to specify ^2.1.1.

Strictly speaking, I think ^2.0.0 is "correct" because this update is just for package dependency and USB camera things are side-effect. Which means there are no change related to USB camera in actfw-jetson but user can now use actfw-jetson with actfw-core=2.1.1 which enable USB camera in 64 bit environments.